include/cudaq/Optimizer/Transforms/Passes.td

```diff
@@ -299,79 +296,4 @@ def UnwindLowering : Pass<"unwind-lowering", "mlir::func::FuncOp"> {
   let constructor = "cudaq::opt::createUnwindLoweringPass()";
 }
 
-def OpCancellation : Pass<"qtx-op-cancellation", "qtx::CircuitOp"> {
-  let summary = "Remove pairs of adjacent adjoint operations.";
-  let description = [{
-    This pass performs basic quantum operation cancellation: it traverses
-    a 'circuit' and removes pairs of adjacent adjoint operations.  This
-    optimization is purely structural, and its effectiveness is highly
-    dependent on canonicalization.
-
-    Note: This is a low-effort pass that only take into account commutations
-    between operations that share the a subset of controls.
-  }];
-  let constructor = "cudaq::opt::createOpCancellationPass()";
-}
-
-def OpDecomposition: Pass<"qtx-op-decomposition", "qtx::CircuitOp"> {
-  let summary = "Remove pairs of adjacent adjoint operations.";
-  let description = [{
-    Decomposition is the process of _systematically_ breaking down
-    high-level instruction (operations) into a sequence of lower level ones.
-    We emphasize "systematically" because it is the characteristic that
-    differentiates decompostion from synthesis.  When doing deomposition, we
-    generate the sequence of lower-level instructions by applying some
-    construction rule(s).  Decomposition techniques are faster than synthesis
-    and produce predictable results, i.e., we know in advance the cost in number
-    of low-level instructions and in the number of qubits.
-
-    NOTE: The implementation still preliminary.
-  }];
-  let constructor = "cudaq::opt::createOpDecompositionPass()";
-}
-
-def SplitArrays: Pass<"qtx-split-arrays", "qtx::CircuitOp"> {
```

Comment:
A pass similar to this probably needs to be implemented in the unified dialect, right?